noir_stdlib/src/meta/typ.nr

```diff
@@ -1,4 +1,13 @@
 use crate::cmp::Eq;
+use crate::option::Option;
+
+impl Type {
+    #[builtin(type_as_integer)]
+    fn as_integer(self) -> Option<(bool, u8)> {}
+
+    #[builtin(type_is_field)]
+    fn is_field(self) -> bool {}
```

Comment:
I noticed none of these built-in methods are documented. Should we start documenting them? (though I know that for now we'd also need to replicate the docs in an `.md`). Or maybe we'll do that once "all" of the builtins are done?

Reply:
You can document them if you want. We'll need to do a very large documentation pass when we want to stabilize all the metaprogramming stuff.